packages/playground/src/react/standalone.tsx

```diff
@@ -39,3 +40,22 @@ export async function renderReactPlayground(config: ReactPlaygroundConfig) {
   const root = createRoot(document.getElementById("root")!);
   root.render(app);
 }
+
+export function createStandalonePlaygroundStateStorage(): StateStorage<PlaygroundSaveData> {
+  const stateStorage = createUrlStateStorate<PlaygroundSaveData>({
+    content: {
+      queryParam: "c",
+      compress: "lz-base64",
+    },
+    sampleName: {
+      queryParam: "sample",
+    },
+  });
+
+  return {
+    load: stateStorage.load,
+    save(data: PlaygroundSaveData) {
+      stateStorage.save(data.sampleName ? { sampleName: data.sampleName } : data);
```

Comment:
seems to not actually work when using cmdOrCtrl+s but works when clicking the button